content/zh-cn/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -164,7 +184,10 @@ that are most appropriate for your configuration, do the following:
       kubectl label --dry-run=server --overwrite ns --all \
       pod-security.kubernetes.io/enforce=baseline
       ```
-      <!-- The output is similar to this: -->
+
+      <!--
+      The output is similar to:
+      :-->
```

Comment:
```suggestion
      -->
```
This line has problem.

<img width="711" alt="image" src="https://user-images.githubusercontent.com/79828097/224879195-0201fdd6-12e6-4fe0-8616-860e63621e4c.png">